.gitignore

```diff
@@ -6,3 +6,4 @@ node_modules
 .env
 .env.*
 !.env.example
+supabase/
```

Comment:
looks like this should be committed to VCS so we can run db migrations and stuff during CI for prod.
```suggestion

```

Reply:
Added in [98728d5](https://github.com/johnverrone/bunkie-bois/pull/8/commits/98728d5bdbdb3656c8a191f08b2ae393c2fe3bdd)